bin/propolis-standalone/src/config.rs

```diff
@@ -43,6 +44,133 @@ pub fn block_backend(
             let creg = ChildRegister::new(&be, Some(path.to_string()));
             (be, creg)
         }
+        "crucible" => {
+            info!(log, "Building a crucible VolumeConstructionRequest from options {:?}", be.options);
```

Comment:
Assuming the crucible feature is made optional, move this into a function gated with a `cfg()` conditional?